RNTester/ComponentRegistry.cpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#ifdef FABRIC_ENABLED
+#include <react/uimanager/ComponentDescriptorFactory.h>
+#include <react/uimanager/ComponentDescriptorRegistry.h>
+#include <react/uimanager/ContextContainer.h>
+
+#include <react/config/ReactNativeConfig.h>
+#include <react/components/image/ImageComponentDescriptor.h>
+#include <react/components/scrollview/ScrollViewComponentDescriptor.h>
+#include <react/components/rncore/ComponentDescriptors.h>
+#include <react/components/text/ParagraphComponentDescriptor.h>
+#include <react/components/text/RawTextComponentDescriptor.h>
+#include <react/components/text/TextComponentDescriptor.h>
+#include <react/components/view/ViewComponentDescriptor.h>
+#include <react/components/slider/SliderComponentDescriptor.h>
+
+namespace facebook {
+  namespace react {
+    
+    /**
```

Comment:
styling nit, we usually don't indent for namespace stuffs, so it should just be:

```
namespace facebook {
namespace react {

/**
 ...
 */
ComponentRegistryFactory ... {
}

} // namespace react
} // namespace facebook